OptimizelySDK/OptimizelyUserContext.cs

```diff
@@ -56,9 +58,19 @@ public OptimizelyUserContext(Optimizely optimizely, string userId, UserAttribute
             Attributes = userAttributes ?? new UserAttributes();
             ForcedDecisionsStore = forcedDecisionsStore ?? new ForcedDecisionsStore();
             UserId = userId;
+            QualifiedSegments = new List<string>();
```

Comment:
Use Array if it's constant.

Reply:
Using [Java SDK as reference](https://github.com/optimizely/java-sdk/blob/8bd6ea5af43120ae1928d00caa44aca67ea80f68/core-api/src/main/java/com/optimizely/ab/OptimizelyUserContext.java#L56) (b/c I can't read Swift yet), I think we want this to not be constant. ...I think